lisa/tools/lscpu.py

```diff
@@ -104,6 +107,36 @@ def get_core_count(self, force_run: bool = False) -> int:
 
         return self._core_count
 
+    def get_thread_per_core_count(self, force_run: bool = False) -> int:
```

Comment:
What's the reason it needs the force run? The value should be fixed, is it right?

Reply:
if it is used in resize vm size, then it will be different.